machine-learning/full-stack-platform-engineer.md

```diff
@@ -0,0 +1,41 @@
+---
+authors: @ltbringer
+groups: product
+teams: platform-ml
+level: L1 L2 L3
```

Comment:
I realized that we were not using proper formatting for YAML frontmatter. I will merge this and fix.